prototype2/common/Socket.cpp

```diff
@@ -79,15 +98,19 @@ void Socket::setLocalSocket(const char *ipaddr, int port) {
 
   int ret = inet_aton(ipaddr, &localSockAddr.sin_addr);
   if (ret == 0) {
-    LOG(IPC, Sev::Error, "invalid ip address {}", ipaddr);
-    throw std::runtime_error("setLocalSocket() - invalid ip");
+    std::string msg;
+    msg = fmt::format("setLocalSocket() - invalid ip address {}", ipaddr);
```

Comment:
Feel free to do:
```
auto Msg = fmt::format("setLocalSocket() - invalid ip address {}", ipaddr);
```
Also, see other occurrences of declaration and assignment in this pull request.